packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -163,6 +164,12 @@ module.exports = (themeOptions = {}) => {
        */
       'gatsby-plugin-sharp',
       'gatsby-plugin-react-helmet',
+      pluginOptions.enableCanonicalUrls && {
+        resolve: 'gatsby-plugin-react-helmet-canonical-urls',
+        options: {
+          siteUrl: `https://${productionHostname}`,
```

Comment:
```suggestion
          siteUrl: `https://${productionHostname}`,
          noTrailingSlash: true,
          noHash: true,
          noQueryString: true,
```
https://www.gatsbyjs.com/plugins/gatsby-plugin-react-helmet-canonical-urls/#how-to-use

Esp. for the release notes overview we must not have query strings in the URL.